app/structures/adjacency_list.py

```diff
@@ -245,6 +255,13 @@ def remove_vertex(self, vertex: int):
                 v.remove(vertex)
 
         self.graph.pop(vertex)
+        
+    def generate_pagerank_graph(self, n: int):
```

Comment:
We have generate methods in graph_utils file. Also generator should return new object, yours just
complements the properties